core/run.py

```diff
@@ -225,5 +238,42 @@ def main():
         executor.start_polling(dp, skip_updates=True)
 
 
+def run_agent():
+    raise NotImplementedError
+
+
+def run_service():
+    service_name = args.service_name
+    gateway_config = get_service_gateway_config(service_name)
+    service_config = gateway_config['service']
+
+    formatter = service_config['formatter']
+    connector_type = service_config['protocol']
+    connector_cls = connectors_map[connector_type]
+    connector = connector_cls(service_config=service_config, formatter=formatter)
+
+    transport_type = gateway_config['transport']['type']
+    gateway_cls = gateways_map[transport_type]['service']
+    _gateway = gateway_cls(config=gateway_config, to_service_callback=connector.send_to_service)
+
+    loop = asyncio.get_event_loop()
+    loop.run_forever()
+
+
+def run_channel():
+    raise NotImplementedError
+
+
+def main():
+    if MODE == 'default':
+        run_default()
+    elif MODE == 'agent':
+        run_agent()
+    elif MODE == 'service':
+        run_service()
+    elif MODE == 'channel':
+        run_channel()
```

Comment:
This section doesn't look good. It should not separate 'default' and other things.

Reply:
decided to leave in this PR